### PR TITLE
Rd 3755 document agent install provided

### PR DESCRIPTION
**Pull request by glukhman**

### CODE REVIEW

File: content/install_maintain/agents/installation.md [hunk 1]
```diff
@@ -130,24 +130,30 @@ every agent.
    node instance, or form the deployment logs:    
    ![Script in logs]( /images/manager/agent_installation/logs_script.webp )
 1. Login to the agent's VM.
+1. Prepare the agent directory
+   ```shell
+   sudo mkdir -p /opt/cloudify-agent-6.3.0-ga/
+   sudo chown $(whoami) /opt/cloudify-agent-6.3.0-ga/
```
Comment (geokala) [inline content/install_maintain/agents/installation.md:136]: Why set it to be owned by the user? Shouldn't it be root?

File: content/install_maintain/agents/installation.md [hunk 2]
```diff
@@ -130,24 +130,30 @@ every agent.
...
+   mkdir -p /opt/cloudify-agent-6.3.0-ga/container_nwxqiu/cloudify/ssl
```
Comment (geokala) [inline content/install_maintain/agents/installation.md:137]: Anywhere you're using the agent's identified (container_nwxqiu), you should probably be using a stirng that is noted at the beginning?

File: content/install_maintain/agents/installation.md [hunk 3]
```diff
@@ -130,24 +130,30 @@ every agent.
...
    ("container_nwxqiu" in the above example):
    ```shell
-   tar xzfv centos-core-agent.tar.gz --strip=1 -C ~centos/container_nwxqiu
+   tar xzfv centos-core-agent.tar.gz --strip=1 -C /opt/cloudify-agent-6.3.0-ga/container_nwxqiu
    ```
-1. Download the configuration script using the like from step 4:
+1. Download the configuration script using the link from step 4:
    ```shell
-   wget --no-check-certificate --user admin --password admin https://10.239.0.207:53333/resources/cloudify_agent/3631bbc3-fc37-4a9f-9db3-81adb2d8e182.sh
+   wget --no-check-certificate --user admin --password admin https://10.239.x.y:53333/resources/cloudify_agent/3631bbc3-fc37-4a9f-9db3-81adb2d8e182.sh
```
Comment (geokala) [inline content/install_maintain/agents/installation.md:146]: Just use RFC5737 compliant IPs- e.g. https://192.0.2.123:53333/etc/etc

**Review by geokala — APPROVED** [review]

File: content/install_maintain/agents/installation.md (hunk 1 above)
Comment (glukhman) [reply content/install_maintain/agents/installation.md:136]: otherwise the script won't work, even with sudo

Comment (geokala) [reply content/install_maintain/agents/installation.md:136]: That doesn't sound right- I thought we set that dir to be owned as root when we installed agents. How does the script fail?

If we don't do that then presumably trying to reinstall an agent as a different user will result in fire and sadness.

Comment (geokala) [reply content/install_maintain/agents/installation.md]: After further discussion, we're going to leave this part of the instructions alone for the moment as it should only have impact if installing multiple agents on the same machine using this approach.

**Review by geokala — APPROVED** [review]

**Pull request merged**